pages/datarooms/index.tsx

```diff
@@ -35,6 +38,10 @@ export default function DataroomsPage() {
   const isTrialDatarooms = trial === "drtrial";
   const canCreateUnlimitedDatarooms =
     isDatarooms || (isBusiness && numDatarooms < limitDatarooms);
+  
+  useEffect(()=>{
+    if(trial == null) router.push('/documents')
+  },[trial])  
```

Comment:
@nidhish-srivastava this might have unindented consequences. 

for example it would push everyone who can view the datarooms page on a plan that supports it (business, datarooms) to be redirected to /documents because in their case `trial == null` is true

Reply:
Yes you are right